package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "node-openssl",
+  "version": "0.1.0",
+  "main": "main.js",
+  "private": true,
+  "gypfile": true,
+  "scripts": {
+    "start": "node-gyp configure build && node module.js",
```

Comment:
its weird to have a start script for a lib like this

Reply:
It doesn't work it needs deleting